dev/com.ibm.ws.concurrent.persistent_fat/test-applications/schedtest/src/web/SchedulerFATServlet.java

```diff
@@ -80,6 +84,11 @@
      */
     private static final String SUCCESS_MESSAGE = "COMPLETED SUCCESSFULLY";
 
+    /**
+     * Indicates if the database for the persistent executor is Derby.
+     */
+    private boolean isDerby;
```

Comment:
You could instead use: 
```
private final boolean isDerby = System.getProperty("fat.bucket.db.type", "Derby").compareToIgnoreCase("Derby") == 0;
```
And avoid making a connection in the `init` method

Reply:
That's a nice improvement. I'll update it.